### PR TITLE
docs: adjust new schedule

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- https://github.com/eggjs/egg-schedule/pull/23

### CODE REVIEW

**Comment by atian25** [comment]
这个也看看，配套文档

File: docs/source/zh-cn/basics/schedule.md [hunk 1]
```diff
@@ -147,25 +153,28 @@ module.exports = app => {
...
+    }
+  }
+  agent.schedule.use('cluster', ClusterStrategy);
 };
 ```
 
-schedule handler 接受两个个参数：
+`ScheduleStrategy` 基类提供了：
 
 - `schedule` - 定时任务的属性，disable 和 immediate 是默认统一支持的，其他配置可以自行解析。
```
Comment (atian25) [inline docs/source/zh-cn/basics/schedule.md]: 这里有问题，我改改

File: docs/source/zh-cn/basics/schedule.md [hunk 2]
```diff
@@ -147,25 +153,28 @@ module.exports = app => {
...
 ```
 
-schedule handler 接受两个个参数：
+`ScheduleStrategy` 基类提供了：
 
-- `schedule` - 定时任务的属性，disable 和 immediate 是默认统一支持的，其他配置可以自行解析。
-- `sender` - 发消息通知 worker 执行 task 的方法。
-  - `sender.one()` - 随机通知一个 worker 执行 task。
-  - `sender.all()` - 通知所有的 worker 执行 task。
+- `schedule` - 定时任务的属性，`disable`  是默认统一支持的，其他配置可以自行解析。
```
Comment (atian25) [inline docs/source/zh-cn/basics/schedule.md]: 干掉了 `immediate`

看看没啥问题就合并吧

**Comment by popomore** [comment]
更新了下

File: docs/source/zh-cn/basics/schedule.md [hunk 3]
```diff
@@ -16,13 +16,37 @@ title: 定时任务
...
+class UpdateCache extends Subscription {
   // 通过 schedule 属性来设置定时任务的执行间隔等配置
+  static get schedule() {
+    return {
+      interval: '1m', // 1 分钟间隔
+      type: 'all', // 指定所有的 worker 都需要执行
+    };
+  }
+
+  // task 是真正定时任务执行时被运行的函数，第一个参数是一个匿名的 Context 实例
```
Comment (atian25) [inline docs/source/zh-cn/basics/schedule.md]: `task` -> `subscribe`

File: docs/source/zh-cn/basics/schedule.md [hunk 4]
```diff
@@ -34,6 +58,11 @@ module.exports = {
 
 编写完这个文件就意味着一个定时任务定义完成了。这个定时任务会在每一个 Worker 进程上每 1 分钟执行一次，将远程数据请求回来挂载到 `app.cache` 上。
 
+### 任务
+
+- `task` 函数或 `subscribe` 方法支持 `generator function / async function`。
```
Comment (atian25) [inline docs/source/zh-cn/basics/schedule.md]: `函数` 和 `方法` 都干掉或者统一一个?

Comment (popomore) [reply docs/source/zh-cn/basics/schedule.md]: 去掉吧，这两个是两个意思，没法统一。

**Comment by atian25** [comment]
+1

**Pull request merged**